langfuse/client.py

```diff
@@ -2817,7 +2862,7 @@ class DatasetClient:
     created_at: dt.datetime
     updated_at: dt.datetime
     items: typing.List[DatasetItemClient]
-    runs: typing.List[str]
+    runs: typing.List[str] = []  # deprecated
```

Comment:
semi-breaking, there is a dedicated list endpoint. options: include fetching pages in get_dataset() to be strictly non-breaking but with increased overhead, return empty array, remove runs from attributes.